apis/r/tests/testthat/test-11-reopen.R

```diff
@@ -54,6 +54,15 @@ test_that("`reopen()` works on arrays", {
     )
     expect_true(arr$is_open(), info = is_open)
 
+    # Test reopen("DELETE")
+    expect_invisible(arr$reopen("DELETE"), label=lab)
```

Comment:
Style suggestion: R code tends to include spaces around `=` in function calls (also for assignment, but using `=` for assignment instead of `<-` is frowned upon)
```suggestion
    expect_invisible(arr$reopen("DELETE"), label = lab)
```